doc/source/index.rst

```diff
@@ -10,22 +10,41 @@ and provides linear scalability for a growing environment.
 
 Features of |PXC| include:
 
-* **Synchronous replication**:
+* **Synchronous replication**
+
   Data is written to all nodes simultaneously,
   or not written at all if it fails even on a single node.
 
-* **Multi-master replication**:
+* **Multi-master replication**
+
   Any node can trigger a data update.
 
-* **True parallel replication**:
+* **True parallel replication**
+
   Multiple threads on slave performing replication on row level.
 
-* **Automatic node provisioning**:
+* **Automatic node provisioning**
+
   You simply add a node and it automatically syncs.
 
-* **Data consistency**:
+* **Data consistency**
+
   No more unsynchronized nodes.
 
+* **PXC Strict Mode**
+
+  Avoids the use of experimental and unsupported features.
+
+* **Configuration script for ProxySQL**
+
+  Percona provides a ProxySQL package with the ``proxysql-admin`` tool
+  that automatically configures |PXC| nodes.
+
+* **Automatic configuration of SSL encryption**
+
+  |PXC| includes the ``pxc-encrypt-cluster-traffic`` variable
+  that enables automatic configuration of SSL encrytion.
+
```

Comment:
@kbauskar which one?
https://www.percona.com/blog/2017/04/19/performance-improvements-percona-xtradb-cluster-5-7-17/
https://www.percona.com/blog/2017/04/19/how-we-made-percona-xtradb-cluster-scale/
Link both posts?